bluemira/equilibria/fem_fixed_boundary/fem_magnetostatic_2D.py

```diff
@@ -298,7 +110,7 @@ def __init__(
         I_p: Optional[float] = None,
         R_0: Optional[float] = None,
         B_0: Optional[float] = None,
-        p_order: int = 3,
+        p_order: int = 2,
```

Comment:
@ivanmaione There's a real argument here to removing all the inputs from p_prime to B_0 from the __init__, seeing as we presently have to use `.set_profiles` after the mesh is set, and to enable different ordering of these calls could get messy.

Like, right now, if you set p_prime and ff_prime in the init, but don't use set_profiles before solve, you get an error.